spring-cloud-stream-binder-test/src/main/java/org/springframework/cloud/stream/binder/AbstractBinderTests.java

```diff
@@ -521,12 +528,17 @@ public void testSendPojoReceivePojoKryoWithStreamListener() throws Exception {
 				AbstractBinderTests.class, "echoStation", Station.class);
 
 		Binder binder = getBinder();
-		DirectChannel moduleOutputChannel = createBindableChannel("output",
-				new BindingProperties());
-		DirectChannel moduleInputChannel = createBindableChannel("input",
-				new BindingProperties());
+
+		PP producerProperties = createProducerProperties();
+
+		BindingProperties producerBindingProperties = createProducerBindingProperties(producerProperties);
+
+		DirectChannel moduleOutputChannel = createBindableChannel("output", producerBindingProperties);
+		DirectChannel moduleInputChannel = createBindableChannel("input", producerBindingProperties);
```

Comment:
This looks wrong; input channels use `consumerBindingProperties`.

(multiple places).